pyproject.toml

```diff
@@ -31,6 +31,7 @@ dependencies = [
     "langchain>=0.0.311",
     "every-ai>=1.1.1",
     "aiohttp>=3.9.0b0; python_version >= '3.12'",
+    "channels>=3.0.5",
```

Comment:
suggestion: make this an optional dependency. Maybe

```toml
[project.optional-dependencies]
# ...
sse = [
    "channels>=3.0.5",
]
``` 

or `channels = `

I missed any discussions that happened around this, but my stance is we should keep core dependencies to a minimum as much as possible. i.e. if I am going to use wagtail-vector-index` for similarity (related content) or non-streaming Q&A-like functionality, why would I need channels?